bootloader/wscript

```diff
@@ -665,22 +665,48 @@ def configure(ctx):
 
     ## setup DEBUG environment
     ctx.setenv('debug', basic_env)  # Ensure env contains shared values.
+    set_win_link(ctx.env, False)
     debug_env = ctx.env
     # This defines enable verbose console output of the bootloader.
     ctx.env.append_value('DEFINES', ['LAUNCH_DEBUG'])
     ctx.env.append_value('DEFINES', 'NDEBUG')
 
     ## setup windowed DEBUG environment
     windowed('debugw', debug_env)
+    set_win_link(ctx.env, True)
 
     ## setup RELEASE environment
     ctx.setenv('release', basic_env)  # Ensure env contains shared values.
     release_env = ctx.env
+    set_win_link(ctx.env, False)
     ctx.env.append_value('DEFINES', 'NDEBUG')
 
     ## setup windowed RELEASE environment
     windowed('releasew', release_env)
-
+    set_win_link(ctx.env, True)
+
+def set_win_link(env, win):
```

Comment:
I'm not convinced, this is a good place for doing this. I understand you need to pass different options for consoled and windowed, but there already is `windowed()` to set the values for the later case. So the default should be to set the "console"-options (around line 621, where LINKFLAGS are already set) and change them in `windowed()` ([`env._get_list_value_for_modification()`](https://waf.io/apidocs/ConfigSet.html#waflib.ConfigSet.ConfigSet._get_list_value_for_modification) might be helpful).

Also this should be **well** documented, what **exactly* it does and that this is required for Windows XP compatibility.